modules/ROOT/pages/deployment/general/ocis-init.adoc

```diff
@@ -6,36 +6,70 @@
 
 {description}
 
-== When to Use
+== Deployments
 
 In general, the xref:deployment/general/general-info.adoc#initialize-infinite-scale[ocis init] command initializes ocis _for the first run_ and creates an `ocis.yaml` configuration file. See xref:deployment/general/general-info.adoc#configuration-rules[Configuration Rules] for the file location. This command is helpful if you do not provide the necessary settings manually, but some rules apply.
 
-* When using the xref:deployment/binary/binary-setup.adoc[Binary Setup], the command *is recommended* to be run manually once before first usage, though you can also fully configure the initial setup manually.
-+
---
-When running `ocis init`, a random admin password will be printed to the shell for first login. Though the password can be changed afterwards in the UI, it is possible to define it right from the start when initializing, the password below is just an example:
+=== Binary Setup
+
+When using the xref:deployment/binary/binary-setup.adoc[Binary Setup], the command *is recommended* to be run manually once before first usage, though you can also fully configure the initial setup manually.
 
+* Following command line parameters or their equivalent environment variables can be defined to configure the `ocis init` command. When using environment variables the following structure is used, multiple variables are allowed:
++
 [source,bash]
 ----
-IDM_ADMIN_PASSWORD=jvQN4XW2q&bjPAj7 \
+<variable1>=<value1> \
+<variable2>=<value2> \
+...
 ocis init
 ----
+
+* `--admin-password` or `--ap` +
+`ADMIN_PASSWORD` or `IDM_ADMIN_PASSWORD`, value=password
++
+When running `ocis init`, a random admin password will be printed to the shell for first login. Though the password can be changed afterwards in the UI, it is possible to define it right from the start when initializing.
+
+* `--insecure` +
+`OCIS_INSECURE`, value=true|false
++
+This allows to use transport security, but disables certificate verification. Useful with self signed certificates to avoid certificate warnings. If set, the value will also be written to the config file. In such a case, when calling `ocis server`, it is not necessary to set the environment variable again on each start.
```

Comment:
```suggestion
This allows to use transport security, but disables certificate verification. Useful with self-signed certificates to avoid certificate warnings. If set, the value will also be written to the config file. In such a case, when calling `ocis server`, it is not necessary to set the environment variable again with each start.
```